<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>780</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CLOSE</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>881</width>
      <height>771</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="scan_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
    <widget class="QComboBox" name="serial_comboBox">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="speed_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="open_btn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>670</y>
       <width>861</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="start_serial_btn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_serial_btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recording_btn">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>REC</string>
     </property>
    </widget>
    <widget class="PlotWidget" name="acc_x_plot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>acc_x</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="stoprecording_btn">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>STOPREC</string>
     </property>
    </widget>
    <widget class="PlotWidget" name="gyro_x_plot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>gyro_x</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="mag_x_plot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>mag_x</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="btn_1_plot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>btn_1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="encoder_plot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>570</y>
       <width>421</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>encoder</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="battery_plot" native="true">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>570</y>
       <width>431</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>battery</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="acc_y_plot" native="true">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>acc_y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="gyro_y_plot" native="true">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>170</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>gyro_y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="mag_y_plot" native="true">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>270</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>mag_y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="btn_2_plot" native="true">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>370</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>btn_2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="acc_z_plot" native="true">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>70</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>acc_y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="gyro_z_plot" native="true">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>170</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>gyro_z</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="mag_z_plot" native="true">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>270</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>mag_z</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="btn_3_plot" native="true">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>370</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>btn_3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MindCube - GUI</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="PlotWidget" name="btn_4_plot" native="true">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>370</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>btn_4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Time stamp:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Seq number:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="timestamp_label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="seq_label">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>40</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="PlotWidget" name="joystick_x_plot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>421</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>joystick_x</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="joystick_y_plot" native="true">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>470</y>
       <width>431</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>joystick_y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
